db/re/instance_db.yml

```diff
@@ -0,0 +1,267 @@
+# This file is a part of rAthena.
+#   Copyright(C) 2019 rAthena Development Team
+#   https://rathena.org - https://github.com/rathena
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+###########################################################################
+# Renewal Instance Database
+###########################################################################
+#
+# Instance Settings
+#
+###########################################################################
+# - Id              Unique Instance ID.
+#   Name            Unique Instance Name.
+#   LimitTime       Total lifetime of instance in seconds.
+#   IdleTimeOut     Time before an idle instance is destroyed in seconds. (Default: 300)
+#   EnterMap        Map Name where players start.
+#   EnterX          X Coordinate where players start.
+#   EnterY          Y Coordinate where players start.
+#   AdditionalMaps  List of maps that are part of an instance.
+###########################################################################
+
+Header:
+  Type: INSTANCE_DB
+  Version: 1
+
+Instances:
```

Comment:
is there should be "Body" not "Instances" ?

Reply:
Correct. I forgot to convert those after merging to the new YAML class! Thanks!